packages/cardpay-sdk/sdk/prepaid-card/base.ts

```diff
@@ -33,6 +33,8 @@ const TIMEOUT = 1000 * 60 * 5;
 // stay below the AML limit of $10,000 USD.
 export const MAXIMUM_PAYMENT_AMOUNT = 10000 * 100;
 export const MAX_PREPAID_CARD_AMOUNT = 10;
+const REWARD_PROGRAM_REGISTRATION_FEE_IN_SPEND = 500;
+const REWARDEE_REGISTRATION_FEE_IN_SPEND = 500;
```

Comment:
are these constants available on chain? if so we should refer to those in case chris decides to change the fees